Makefile

```diff
@@ -136,7 +136,7 @@ test_sim_modules:
 
 test_sim_gaia_fast:
 	@echo "Running full Gaia simulation. This may take several minutes..."
-	@go test ./cmd/gaia/app -run TestFullGaiaSimulation -SimulationEnabled=true -SimulationNumBlocks=200 -timeout 2h
+	@go test ./cmd/gaia/app -run TestFullGaiaSimulation -SimulationEnabled=true -SimulationNumBlocks=200 -timeout 24h
```

Comment:
This shouldn't be fast. Perhaps make a test_sim_gaia_release, and have that run only on merges to master